### PR TITLE
[[Bug 19597]][docs] Open Socket summary edit

### PR DESCRIPTION
**Pull request by seaniepie**
• Update to summary to include UDP
• Added glossary links for UDP & datagram

### CODE REVIEW

**Comment by livecodepanos** [comment]
Thanks for the PR @seaniepie.

The docs updates should be submitted against `develop-9.0` branch. Would you mind rebasing this PR onto `develop-9.0` branch? I would do it myself from the web interface, but in this case it is not possible.

**Comment by livecodepanos** [comment]
@seaniepie I will post the same comment as in https://github.com/livecode/livecode/pull/6720, for anyone that has not seen it.

Since this branch (`seaniepie:bugfix-19597`) was originally branched off from develop, it contains some changes that exist only in develop. So if this PR is rebased against develop-9.0 from the GitHub web interface, it will contain not only your changes in the documentation but also all the other changes that exist in develop and not in develop-9.0.

Long story short, for documentation PRs, we have to:

Branch off from develop-9.0
Submit a PR against develop-9.0

Just for this PR and https://github.com/livecode/livecode/pull/6720 I will rebase them back into develop, and once merged I will cherry-pick the changes back into develop-9.0.

Sorry for the confusion :)

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok  f6be939

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok f6be939ab271fef030bc86705ee6cbbd91b4ea37

**Comment by livecode-vulcan** [comment]
😎 test success f6be939ab271fef030bc86705ee6cbbd91b4ea37

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/128)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/120)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/120)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/120)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/120)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1642)
* try-community-universal-ios-iphoneos12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.0/builds/24)
* try-community-universal-ios-iphonesimulator12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.0/builds/24)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1672)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1665)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1662)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1698)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1639)
